### ISSUE TITLE
does the power management work? 

### ISSUE THREAD

**Issue by kukat** [xlanor/asus-z390i-opencore#1]
I have the same specs, Asus z390i + 9900k
but I can't check if the power management is working, when I open Intel Power Gadget I got this screen, follow the instruction, restart, the same thing happens
![image](https://user-images.githubusercontent.com/107503/101258951-dc756000-3760-11eb-9218-e3633eb6ce71.png)

**Comment by xlanor** [comment]
No issues on my end.

![Screenshot 2020-12-06 at 1 42 27 PM](https://user-images.githubusercontent.com/29213366/101272630-e3d05400-37c8-11eb-81b9-0cacb43eea7b.png)

**Comment by kukat** [comment]
hmmm, going to check my bios settings...

original BT card do you mean the intel 9560? i've replaced it with dw1560. but the BT experience sucks... trackpad is working but airpods mic quality is poor. 😞

**Comment by xlanor** [comment]
Have you tried rolling it back to 3.5.0 (IPG)? there's some talk on some hackintoshing threads I saw last night about it helping.

How's sidecar on your side? I've replaced it with 1560, and I still cant get sidecar to work.

My airpods also work, but I generally use a cheap usb microphone instead for doing calls. I don't like airpod's mic. 

BT audio has a slight delay for me, but other than that its pretty smooth. I do have issues with waking though.
